### ISSUE TITLE
[IMRPOVEMENTS]: Rotating Objects becomes incredibly confusing after rotating so much

### ISSUE THREAD

**Issue by AkkozaDevelops** [typical-developers/oaklands-feedback#23]
### Roblox Username

Akkoza

### What do you think can have quality-of-life improvements?

Drag System

### How do you think things can be improved?

Reset the rotation to reduce confusion

**Comment by ManancialGD** [comment]
Yes!!! I was going to say that, if you turn it to the right 90º, when you hold "w", it does not roll frontwards and backwards, it rolls sideways, and it's hard, because I want to roll to the direction of my character, but it rolls to another direction.

**Comment by AkkozaDevelops** [comment]
Will spend some time possibly tomorrow into figuring out how to make this improvement.
